src/commcare_cloud/ansible/roles/nginx/templates/site.j2

```diff
@@ -47,6 +47,13 @@ proxy_cache_path {{ cache.path }} levels=1:2 keys_zone={{ cache.name }}:10m
 {% endfor %}
 
 server {
+{% if trusted_proxies %}
+  real_ip_header X-Forwarded-For;
```

Comment:
One note: The PR description doesn't specifically say that this sets the forwarded for header, just that it adds the proxies, so it might be confusing for someone to run across.

Reply:
OK I added some comments here and I'll update the PR description